ethcore/sync/src/tests/helpers.rs

```diff
@@ -210,7 +210,7 @@ pub struct EthPeer<C> where C: FlushingBlockChainClient {
 	pub snapshot_service: Arc<TestSnapshotService>,
 	pub sync: RwLock<ChainSync>,
 	pub queue: RwLock<VecDeque<TestPacket>>,
-	pub private_tx_handler: Arc<PrivateTxHandler>,
+	pub private_tx_handler: Arc<SimplePrivateTxHandler>,
```

Comment:
For tests this abstraction doesn't make a lot of sense, but made access to the internal fields of SimplePrivateTxHandler more difficult